android/src/main/java/com/tailscale/ipn/MainActivity.kt

```diff
@@ -64,62 +59,49 @@ class MainActivity : ComponentActivity() {
                             onNavigateToMDMSettings = { navController.navigate("mdmSettings") },
                             onNavigateToManagedBy = { navController.navigate("managedBy") })
 
-                    composable("main") {
-                        MainView(
-                            viewModel = MainViewModel(manager.model, manager),
```

Comment:
Notice that we don't construct any view models inside of the MainActivity anymore.